script.js

```diff
@@ -0,0 +1,105 @@
+const speakers = [
+  {
+    name: 'Shri Anurag Singh Thakur',
+    title: "Hon'ble Minister Of Youth Affairs and Sports",
+    image: './assets/anurag.jpg',
+    bio: 'Mr Anurag Singh Thakur is the Union Minister for Information & Broadcasting and Youth Affairs & Sports, Government of India.',
+  },
+  {
+    name: 'Mahendra Singh Dhoni ',
+    title: 'Former Indian cricketer and captain (2004-2019)',
+    image: './assets/Dhoni1234.png',
+    bio: "India's most successful cricket captain, leading the team to victory in multiple tournaments including the 2007 ICC World Twenty20, 2011 Cricket World Cup, and 2013 ICC Champions Trophy.",
+  },
+  {
+    name: 'Sunil Chhetri',
+    title: 'Indian professional footballer',
+    image: './assets/sunil.jpg',
+    bio: 'Popularly known as Captain Fantastic , he holds the second position in terms of scoring most goals in international matches among active players after Cristiano Ronaldo.',
+  },
+  {
+    name: 'Mangte Chungneijang Mary Kom',
+    title: 'Indian Ameature Boxer',
+    image: './assets/marry.jpg',
+    bio: "She's the sole female boxer to win medals in the first seven World Championships, and the only one to win eight World Championship medals.",
+  },
+  {
+    name: 'Neeraj Chopra',
+    title: 'Track and Field Athlete',
+    image: './assets/neeraj.jpg',
+    bio: "He's the current Olympic champion, World Championship silver medalist, and Diamond League champion in Javelin throw, and the first Asian to win Men's Javelin throw Olympic gold.",
+  },
```

Comment:
- [ ] Kindly add one more speaker as suggested earlier, this will make your work a good prototype of the given template. 👍